### PR TITLE
Pdfs multisearchers

### PR DESCRIPTION
**Pull request by jmayntzhusen**
@Shazwazza asked me to add the information he has on multisearchers from the readme here: https://github.com/umbraco/UmbracoExamine.PDF

### CODE REVIEW

File: Reference/Searching/Examine/index.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ Get up and running with Examine straight away with this quick start guide.
 
 Learn how to create your own Lucene indexes using Examine in Umbraco 8.
 
+## [PDF indexing and multisearchers](pdf-index.md)
+
+Learn how to index PDF files in Examine and how to create a multisearcher that searches through both the externalindex and the pdfindex.
```
Comment (sofietoft) [inline Reference/Searching/Examine/index.md]: Wondering whether _externalindex_ and _pdfindex_ should be written in a different way?
Perhaps something like _External Index_ and _PDF Index_? 🤔

File: Reference/Searching/Examine/pdf-index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+versionFrom: 8.0.0
+---
+
+# Pdf indexes and multisearchers
```
Comment (sofietoft) [inline Reference/Searching/Examine/pdf-index.md]:
```suggestion
# PDF indexes and multisearchers
```

File: Reference/Searching/Examine/pdf-index.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+If you want to index pdfs and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
```
Comment (sofietoft) [inline Reference/Searching/Examine/pdf-index.md]:
```suggestion
If you want to index PDF files and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
```

File: Reference/Searching/Examine/pdf-index.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## Installation
+
+Install with Nuget:
+`Install-Package UmbracoCms.UmbracoExamine.PDF`
+
+You will then have a new Examine index called "PDFIndex" available.
```
Comment (sofietoft) [inline Reference/Searching/Examine/pdf-index.md]: _Examine index_ - really a good one for our style guide.
I think it should be _Examine Index_, but perhaps that's wrong? 🤔

File: Reference/Searching/Examine/pdf-index.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+using(var multiSearcher = new MultiIndexSearcher("MultiSearcher", new IIndex[] {
+    externalIndex,
+    pdfIndex
+}))
+{
+ ...
+};
+```
+
+Or you can register a multi-index searcher with the ExamineManager on startup like:
```
Comment (sofietoft) [inline Reference/Searching/Examine/pdf-index.md]: Is it "multi-index" or "multi index" ?

File: Reference/Searching/Examine/pdf-index.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        _examineManager = examineManager;
+    }
+
+    public void Initialize()
+    {
+        //Get both the external and pdf index
+        if (_examineManager.TryGetIndex(Constants.UmbracoIndexes.ExternalIndexName, out var externalIndex)
+            && _examineManager.TryGetIndex(PdfIndexConstants.PdfIndexName, out var pdfIndex))
+        {
+            //register a multi searcher for both of them
```
Comment (sofietoft) [inline Reference/Searching/Examine/pdf-index.md]: Again, is "multi searcher" one word or two words?
Other places, you've written MultiSearcher, or multisearcher 😅 

Sorry if I'm just confusing this more than necessary!

File: Reference/Searching/Examine/pdf-index.md (hunk 4 above)
Comment (jmayntzhusen) [reply Reference/Searching/Examine/pdf-index.md]: think that is wrong, an index is just a term, like you would say file - not File 🙂

File: Reference/Searching/Examine/pdf-index.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+```cs
+if (_examineManager.TryGetSearcher("MultiSearcher", out var searcher))
+{
+    //TODO: use the `searcher` to search
+}
+```
+
+:::warning
+The implementation of IPdfTextExtractor is PdfSharpTextExtractor in this library which uses PDFSharp to extract the bytes to convert to text but that implementation doesn't deal well with Unicode text which means when some PDF files are read, the result will be 'junk' strings.
```
Comment (jmayntzhusen) [inline Reference/Searching/Examine/pdf-index.md]:
```suggestion
The implementation of IPdfTextExtractor is PdfSharpTextExtractor in this library, which uses PDFSharp to extract the bytes to convert to text. That implementation doesn't deal well with Unicode text which means when some PDF files are read, the result will be 'junk' strings.
```

File: Reference/Searching/Examine/pdf-index.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+:::warning
+The implementation of IPdfTextExtractor is PdfSharpTextExtractor in this library, which uses PDFSharp to extract the bytes to convert to text. That implementation doesn't deal well with Unicode text which means when some PDF files are read, the result will be 'junk' strings.
+
+It is certainly possible to replace the IPdfTextExtractor using your own composer like
+
+`composition.RegisterUnique<IPdfTextExtractor, MyCustomSharpTextExtractor>();`
+
+The iTextSharp library deals with Unicode in a better way but is a paid for license. If you wish to use iTextSharp or another PDF library you can easily swap out the IPdfTextExtractor with your own implementation.
```
Comment (jmayntzhusen) [inline Reference/Searching/Examine/pdf-index.md]:
```suggestion
The iTextSharp library deals with Unicode in a better way but is a paid for license. If you wish to use iTextSharp or another PDF library you can swap out the IPdfTextExtractor with your own implementation.
```

**Pull request merged**